examples/advanced/bionemo/downstream/tap/jobs/central_tap_esm1nv/app1/custom/downstream_flip_tap.yaml

```diff
@@ -46,7 +69,7 @@ model:
       val: tap_valid
       test: tap_test
     sequence_column: "Antibody" # name of column with protein sequence in csv file
-    target_column: ["PSH"] #["3state", "resolved"] # names of label columns in csv file
+    target_column: ["SFvCSP"] #["3state", "resolved"] # names of label columns in csv file
```

Comment:
Sorry, just noticed. The column names shouldn't change. Each app needs to use a different endpoint name for cross-compound training, i.e.

````
patches of surface hydrophobicity (PSH)
patches of positive charge (PPC)
patches of negative charge (PNC)
structural Fv charge symmetry parameter (SFvCSP)
```